automation/features/step-definitions/staking.steps.js

```diff
@@ -48,23 +75,321 @@ When(/^I click on use maximum button$/, () => {
     reverse: false,
     timeoutMsg: "callout was not found",
   });
-  if (browser.getByTestId("associate-more-tokens-btn").isDisplayed()) {
-    browser.getByTestId("associate-more-tokens-btn").click();
-  } else {
-    browser.getByTestId("associate-tokens-btn").click();
-  }
+  if (stakingPage.associateMoreTokensBtn.isDisplayed()) {
+    stakingPage.associateMoreTokensBtn.click();
+  } else stakingPage.associateTokensBtn.click();
   $('[data-testid="associate-radio-wallet"]').click({ force: true });
-  currentAmountOfTokensInWallet = $(".wallet-card__asset-balance").getText();
+  currentAmountOfTokensInWallet = browser
+    .getByTestId("eth-wallet-balance")
+    .getText();
   browser.getByTestId("token-amount-use-maximum").click();
 });
 
 When(
   /^I can see the maximum amount of tokens in my wallet are in the token input box$/,
   () => {
     const noZeroes = currentAmountOfTokensInWallet.toString();
-    const noZeroesFloat = parseFloat(noZeroes);
+    const noZeroesFloat = parseFloat(noZeroes.replace(",", ""));
     expect(browser.getByTestId("token-amount-input").getValue()).toBe(
       String(noZeroesFloat)
     );
   }
 );
+
+Then(/^I can see the validator node list$/, () => {
+  expect(browser.getByTestId("validator-node-list")).toBeDisplayed();
+  const nodeList = $(".node-list").$$('[data-testid="node-list-item"]');
+  expect(nodeList.length).toBeGreaterThan(0);
+});
+
+Then(/^the epoch countdown timer is counting down$/, () => {
+  const currentCountdownTimerText = browser
+    .getByTestId("current-epoch-ends-in")
+    .getText();
+  /* eslint-disable wdio/no-pause */
+  browser.pause(2100); // let some time pass
```

Comment:
```suggestion
  browser.pause(2100);
```

Reply:
Why is this pause needed? Can we not wait for something in the dom?

Reply:
Ditmir will add support for epoch wait check using API which provides determinstic wait time